### PR TITLE
Add proper support for multi-host setup

### PR DESCRIPTION
**Pull request by AlexeyKurakin**

### CODE REVIEW

File: objax/util/util.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 __all__ = ['EasyDict', 'args_indexes', 'dummy_context_mgr', 'ilog2', 'map_to_device', 'positional_args_names',
-           'to_tuple']
+           'to_tuple', 'multi_host_barrier']
```
Comment (david-berthelot) [inline objax/util/util.py:16]: Lete's keep the order alphabetical.

File: objax/util/util.py [hunk 2]
```diff
@@ -71,3 +71,8 @@ def to_tuple(v: Union[Tuple[Number, ...], Number, Iterable], n: int):
         return (v,) * n
     else:
         return tuple(v)
+
+
+def multi_host_barrier():
```
Comment (david-berthelot) [inline objax/util/util.py:76]: Same, move function above so they're alphabetically ordered.

**Review by david-berthelot — CHANGES_REQUESTED** [review]
Some minor nits.

File: objax/util/util.py (hunk 2 above)
Comment (AlexeyKurakin) [reply objax/util/util.py]: Done

**Comment by AlexeyKurakin** [comment]
@david-berthelot , I updated utils.py, please take a look

**Review by david-berthelot — APPROVED** [review]

**Pull request merged**